ui/src/components/ripple-out/support-sections/RippleOutPraySection.js

```diff
@@ -0,0 +1,94 @@
+import { Box, Flex, HStack, Text, VStack } from '@chakra-ui/react';
+import { useState } from 'react';
+import {
+  fontColor,
+  bodyFontSize2,
+  subHeaderFontSize,
+} from '../RippleOutTextStyle';
+
+const SliderDots = (props) => {
+  const { slider, setSlider, length } = props;
+  const dotList = [];
+  for (let i = 0; i < length; i++) {
+    dotList.push(
+      <>
+        <Box
+          border={slider === i ? '8px solid #2A3A58' : '8px solid #B6C1D7'}
+          borderRadius="30px"
+          display={['none', 'flex']}
+          onClick={() => setSlider(i)}
+          cursor="pointer"
+        />
+        <Box
+          border={slider === i ? '6px solid #2A3A58' : '6px solid #B6C1D7'}
+          borderRadius="30px"
+          display={['flex', 'none']}
+          onClick={() => setSlider(i)}
+          cursor="pointer"
+        />
+      </>
+    );
+  }
+  return (
+    <>
+      <HStack w="100%" justifyContent="center" spacing={6}>
+        {dotList}
+      </HStack>
+    </>
+  );
+};
+
+const RippleOutPraySection = ({ prayerData }) => {
+  const data = prayerData.content;
+  const [slider, setSlider] = useState(0);
+  return (
+    <>
+      <Flex>
+        <VStack spacing={8} display="flex" alignItems="flex-start">
+          <Text
+            fontSize={['1.4em', '2.4em']}
+            textStyle="darker_grotesque_black"
+            lineHeight="0.9em"
+          >
+            PRAY FOR THE CAMPAIGN
+          </Text>
+
+          <VStack w="100%" display="flex" alignItems="flex-start" spacing={6}>
```

Comment:
Need a swipe interaction for this according to design. atm the only way for people to switch between slides is to click dots which is not very intuitive